requirements.txt

```diff
@@ -1,3 +1,4 @@
 # Python modules needed for testing
 # (Travis-CI will auto-install them from this file)
 flake8 == 2.5.4
+toml == 0.9.1
```

Comment:
Please also pin this in `homu/init.sls`, and put a comment in both places that mentions the need to keep them in sync. See the [style guide](https://github.com/servo/saltfs/blob/master/STYLE_GUIDE.md#use-pkgs-instead-of-name) for tips.